vms/registration/views.py

```diff
@@ -231,8 +284,12 @@ def post(self, request):
                         org = Organization.objects.create(name=unlisted_org, approved_status=False)
                         org.save()
                         volunteer.organization = org
-
-                    volunteer.reminder_days = 1
+                    if vol_country:
+                        volunteer.country = vol_country
```

Comment:
shouln't there be else where we assign none.

Reply:
I think even this if is not reqired, I have already used if else above.